src/wp-includes/block-supports/duotone.php

```diff
@@ -32,28 +32,30 @@
  * @since 5.8.0
  */
 
+$duotone = new WP_Duotone();
```

Comment:
> There is no new global added here, just a local variable.

This is a global variable.

Any code that runs after this file has been included in `wp-settings.php` will have access to the global `$duotone` variable.

Reply:
Right. WordPress does this in lots of places. I don't see the problem with it....

Reply:
@ajlende @priethor I am going to invert this question, why is static being used in the first place. It is never uncommon in WordPress space to do this? I have heard something about keeping state, but that can be done with a global as well.